### PR TITLE
QA-3560_API_IG_channel

### PR DESCRIPTION
**Pull request by Sashikus**
This PR was created based on subtask https://jira.brandwatch.com/browse/QA-3601

- created gem for IG calls
- verified locally all calls(get, post, put, delete) using 'pry', works fine

### CODE REVIEW

**Comment by Sashikus** [comment]
@jonathanchrisp could you take a look?

File: lib/bwapi/client/projects.rb [hunk 1]
```diff
@@ -120,6 +121,7 @@ def project_twitter_audience_summary(project_id, opts = {})
       include BWAPI::Client::Projects::TwitterQueries
       include BWAPI::Client::Projects::Users
       include BWAPI::Client::Projects::Workflow
+      include BWAPI::Client::Projects::InstagramChannels
```
Comment (jonathanchrisp) [inline lib/bwapi/client/projects.rb]: Is the endpoint called `instagramqueries`? If so we should probably keep to this naming convention.

File: lib/bwapi/version.rb [hunk 2]
```diff
@@ -1,4 +1,4 @@
 # BWAPI Version
 module BWAPI
-  VERSION = '12.0.0'
+  VERSION = '12.3.0'
```
Comment (jonathanchrisp) [inline lib/bwapi/version.rb]: Don't modify the version, we'll do that in a separate PR. Please can you removed this change.

File: lib/bwapi/client/projects.rb (hunk 1 above)
Comment (jonathanchrisp) [inline lib/bwapi/client/projects.rb]: Also, please list this module in alphabetical order.

**Comment by Sashikus** [comment]
@jonathanchrisp fixed , could you take a look once again?

**Comment by jonathanchrisp** [comment]
Looks good to me @Sashikus, @benSlaughter / @leemcgeever can you also take a look?

**Comment by Sashikus** [comment]
please , DON'T MERGE it until IG will be on Live, cc @SirCN

**Comment by Sashikus** [comment]
i'll reopen this PR against stage
